docs.json

```diff
@@ -94,8 +94,15 @@
               "settings/broken-links",
               "settings/github",
               "settings/gitlab",
-              "settings/ci",
-              "settings/preview-deployments"
+              {
+                "group": "Add-Ons",
+                "icon": "puzzle-piece",
```

Comment:
We use a custom SVG in the dashboard for add-ons, which looks kind of like a puzzle piece (and puzzle pieces make sense here)